src/App.js

```diff
@@ -1,24 +1,142 @@
-import activities from './resources/schedule.json'
-import dayjs from "dayjs";
+import { useEffect, useState } from 'react';
+import activities from './resources/schedule.json';
+import dayjs from 'dayjs';
 
 function App() {
+  const [today, setToday] = useState([]);
+  const [tomorrow, setTomorrow] = useState([]);
+  const [nextWeek, setNextWeek] = useState([]);
+  const [future, setFuture] = useState([]);
+
+  const todayDate = new Date();
+  let tomorrowDate = new Date();
+  tomorrowDate.setDate(todayDate.getDate() + 1);
+  const thisWeekEnds = new Date();
+  thisWeekEnds.setDate(todayDate.getDate() + 2);
+
+  let nextWeekStart = new Date(thisWeekEnds + 3);
+  nextWeekStart.setDate(thisWeekEnds.getDate() + 1);
+  // console.log(nextWeekStart);
+
+  let nextWeekEnd = new Date(thisWeekEnds + 1);
+  nextWeekEnd.setDate(nextWeekStart.getDate() + 6);
+
+  useEffect(() => {
+    setToday(
+      activities.filter(
+        (activity) =>
+          dayjs(activity.startTime).format('d MMM') ===
```

Comment:
If the list was longer than what we have now and an an event happened this same day next year, then it'll add that activity to today.